Description: Disable hickory-dns feature and related deps
 There is some problem in hickory-proto.
 .
 trust-dns feature is a leftover when trust-dns was renamed hickory-dns.
Forwarded: not-needed
Last-Update: 2024-11-02
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -116,7 +116,7 @@
     "async-compression?/gzip",
     "dep:tokio-util",
 ]
-hickory-dns = ["dep:hickory-resolver"]
+#hickory-dns = ["dep:hickory-resolver"]
 http2 = [
     "h2",
     "hyper/http2",
@@ -175,7 +175,7 @@
     "dep:tokio-util",
     # "dep:wasm-streams",
 ]
-trust-dns = []
+#trust-dns = []
 zstd = [
     "dep:async-compression",
     "async-compression?/zstd",
@@ -389,10 +389,10 @@
 #version = "0.0.7"
 #optional = true
 
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.hickory-resolver]
-version = "0.24"
-features = ["tokio-runtime"]
-optional = true
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.hickory-resolver]
+#version = "0.24"
+#features = ["tokio-runtime"]
+#optional = true
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies.http-body]
 version = "1"
